sklearn/linear_model/coordinate_descent.py

```diff
@@ -132,17 +132,20 @@ def lasso_path(X, y, eps=1e-3, n_alphas=100, alphas=None,
 
     The Lasso optimization function varies for mono and multi-outputs.
 
-    For mono-output tasks it is::
+    For mono-output tasks it is
 
-        (1 / (2 * n_samples)) * ||y - Xw||^2_2 + alpha * ||w||_1
+    .. math::
+        \\frac{1}{2 * \\mathrm{n\\_samples}} * ||y - Xw||^2_2 + \\alpha * ||w||_1
```

Comment:
Why ? 
On the doc [http://www.sphinx-doc.org/en/stable/ext/math.html], they only recommend:
Keep in mind that when you put math markup in Python docstrings read by autodoc, you either have to double all backslashes, or use Python raw strings (r"raw").